### ISSUE TITLE
Nagging to set severity when severity set to N/A

### ISSUE THREAD

**Issue by PrototypeNM1** [mozilla/relman-auto-nag#1260]
Severity is set to N/A for this issue: https://bugzilla.mozilla.org/show_bug.cgi?id=1746527
After being set to N/A the bot nagged to have severity set.

**Comment by calixteman** [comment]
It's likely because of:
https://github.com/mozilla/relman-auto-nag/blob/56eb129b614c401feff7fa66f1d62518f36cc4c3/auto_nag/scripts/workflow/no_severity.py#L115
The doc for severity:
https://wiki.mozilla.org/BMO/UserGuide/BugFields#bug_severity
and it's mentioned that `N/A` is only for `task` or `enhancement` and here the bug is a `defect`.
If it's really a `defect`, you should set the severity to `S4` (...low or no impact to users).
So for me this "issue" is not a bug.

@PrototypeNM1 does it make sense for you ?

**Comment by PrototypeNM1** [comment]
Makes sense. Think the reasoning is worth noted in the "severity set but invalid" case?

**Comment by calixteman** [comment]
Yep the message in the comment must be more clear.
